airflow/providers/amazon/CHANGELOG.rst

```diff
@@ -24,6 +24,35 @@
 Changelog
 ---------
 
+7.3.0
+.....
+
+Features
+~~~~~~~~
+
+* ``add num rows affected to Redshift Data API hook (#29797)``
+* ``Add 'wait_for_completion' param in 'RedshiftCreateClusterOperator' (#29657)``
+* ``Add Amazon Redshift-data to S3<>RS Transfer Operators (#27947)``
+* ``Allow to specify which connection, variable or config are being looked up in the backend using *_lookup_pattern parameters (#29580)``
+* ``Implement file credentials provider for AWS hook AssumeRoleWithWebIdentity (#29623)``
+* ``Implement custom boto waiters for some EMR operators (#29822)``
+
+Bug Fixes
+~~~~~~~~~
+
+* ``fix code checking job names in sagemaker (#29245)``
+* ``Avoid emitting fallback message for S3TaskHandler if streaming logs (#29708)``
+* ``Fix Amazon ECS Enums (#29871)``
+
+Misc
+~~~~
+
+* ``Impovements for RedshiftDataOperator: better error reporting and an ability to return SQL results (#29434)``
+* ``Use waiters in ECS Operators instead of inner sensors (#29761)``
```

Comment:
I guess we can't do anything when fix introduced for feature/changes in the same release cycle

Reply:
I can hide entries from the change log if needed

Reply:
see https://github.com/apache/airflow/pull/29878/commits/759962e06a4fef20c8754cd9a6b1508b1cc5edd6

Reply:
It shout be another way round: #29761 introduce actual changes, and some kind of bug fixes and I guess better put it in Bug Fixes, and #29871 could be hided because it fix some errors which introduced during #29761 and found by https://aws-mwaa.github.io/open-source/system-tests/dashboard.html

Reply:
done

Reply:
awesome! Thanks!